src/NUnitCommon/nunit.common/TextDisplay/ResultReporter.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Xml;
+
+namespace NUnit.TextDisplay
+{
+    public static class ResultReporter
+    {
+        public static void WriteHeader(ExtendedTextWriter writer)
+        {
+            Assembly entryAssembly = Assembly.GetEntryAssembly() ?? Assembly.GetExecutingAssembly();
+            var versionBlock = FileVersionInfo.GetVersionInfo(entryAssembly.ManifestModule.FullyQualifiedName);
+
+            var header = $"{versionBlock.ProductName} {versionBlock.ProductVersion}";
+
+            var configurationAttributes = entryAssembly.GetCustomAttribute<AssemblyConfigurationAttribute>();
+
```

Comment:
Left in when other code using the AssemblyConfigurationAttribute was removed?
Were you intending to add the `Configuration` to the _header_?

Reply:
Probably a left-over but I'll check.